templates/promoted.html

```diff
@@ -0,0 +1,38 @@
+{% extends "_layout.html" %}
+
+{% block title %}
+  Promoted Snaps — Linux software in the Snap Store
+{% endblock %}
+
+{% block content %}
+  <section class="p-strip is-shallow">
+      <div class="col-12">
+          <h3>Featured snaps</h3>
```

Comment:
These are not featured snaps. Do we need a title here?

Reply:
We won't need a title